File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// NebulumMarsRovers SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class NebulumMarsRoversMakeContext
{
    public static function call(array $ctxmap, ?NebulumMarsRoversContext $basectx): NebulumMarsRoversContext
    {
        return new NebulumMarsRoversContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# NebulumMarsRovers SDK utility: make_context
require_relative '../core/context'
module NebulumMarsRoversUtilities
  MakeContext = ->(ctxmap, basectx) {
    NebulumMarsRoversContext.new(ctxmap, basectx)
  }
end
